.gitignore

```diff
@@ -4,18 +4,15 @@
 .externalToolBuilders
 .idea/
 .settings
-docs/semdom/*.xml
-docs/semdom/semdom lists/*.xml
 node_modules/
 output/chrome/
```

Comment:
```
output/chrome/
```
I don't think this is necessary.  Items in .gitignore are fairly harmless to delete, because the worst thing that can happen is you check-in something you didn't mean to check-in, which is a pain, but not as bad as breaking the application 😁